packages/HorizonUI/Sources/HorizonUI/Sources/Components/NavigationalElements/Overlay/HorizonUI.Overlay.Storybook.swift

```diff
@@ -0,0 +1,80 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2025-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import SwiftUI
+
+public extension HorizonUI.Overlay {
+    struct Storybook: View {
+        @State private var present: Bool = false
```

Comment:
Let's rename this to `isPresented` so it's more inline with the standard SwiftUI naming.